types.cc

```diff
@@ -1811,16 +1811,20 @@ static data_value deserialize_aux(const tuple_type_impl& t, bytes_view v) {
     return data_value::make(t.shared_from_this(), std::make_unique<tuple_type_impl::native_type>(std::move(ret)));
 }
 
-static utils::multiprecision_int deserialize_value(const varint_type_impl&, bytes_view v) {
-    auto negative = v.front() < 0;
+template<FragmentedView View>
+utils::multiprecision_int deserialize_value(const varint_type_impl&, View v) {
+    bool negative = v.current_fragment().front() < 0;
```

Comment:
What if the view is empty? The existing code will survive it.

I don't think we can actually get called here with an empty view, but better to be prepared.

Reply:
> The existing code will survive it.

What do you mean? Calling `front()` on an empty `bytes_view` (aka `std::basic_string_view`) is undefined behaviour.

Reply:
You are right, I withdraw the comment.